scipy/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Compiled Python files
```

Comment:
Why was this file added? It's in the wrong place and the root repo already has a .gitignore file.